### DISCUSSION TITLE
Add `@Ignore` annotation

### DISCUSSION THREAD

**Discussion by jaschdoc** [flix/flix#7206] [Ideas]
Should we add an `@Ignore` annotation for tests like most other test frameworks? It should basically work the same way as in scala test.

- Avoid commenting out the `@Test` annotation, and then avoid the compiler complaining about unused private functions.
- The test framework should then also warn the user that the ignored tests were not run so you are reminded that you have unused code (instead of just silently removing tests if you forget to re-add the test annotation).
- Maybe `@Ignore` can only be used together with `@Test`.

**Comment by magnus-madsen** [comment]
Its called Skip?

**Reply by jaschdoc** [reply]
Alias as `@Ignore` then?

**Reply by magnus-madsen** [reply]
Mja. I prefer not to have aliases. I thought Skip had a clearer meaning and its the same length as Test.
